### DISCUSSION TITLE
[QUESTION] Any way to disable Discord RPC when compiling Kade Engine?

### DISCUSSION THREAD

**Discussion by jelleejamm** [KadeArchive/Kade-Engine#762] [Q&A]
Hi, I've been having some problems compiling Kade Engine due to Discord RPC breaking the compilation. Is there any way to disable it?

**Comment by isakube** [comment]
i think you can comment out any mentions of DiscordRPC classes. i did that with NGio and it worked. it's tedious, but it works.
